app/src/main/java/org/openobservatory/ooniprobe/activity/InfoActivity.java

```diff
@@ -28,6 +28,14 @@
 		version.setText(getString(R.string.version, BuildConfig.SOFTWARE_NAME, BuildConfig.VERSION_NAME));
 	}
 
+	@OnClick(R.id.blog) void onBlogClick() {
+		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.org/blog/")));
+	}
+
+	@OnClick(R.id.reports) void onReportsClick() {
+		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.org/reports/")));
+	}
+
 	@OnClick(R.id.learnMore) void onLearnMoreClick() {
 		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("https://ooni.io/")));
```

Comment:
While you are at it, can you also change the link here to be `ooni.org`?

Reply:
Just didn't know how to handle that. Will the update.